google/cloud/bigtable/tests/admin_integration_test.cc

```diff
@@ -231,6 +233,71 @@ TEST_F(AdminIntegrationTest, CreateListGetDeleteTableTest) {
   EXPECT_EQ(0, table_count);
 }
 
+/// @test Verify that `bigtable::TableAdmin` CheckConsistency works as expected.
+TEST_F(AdminIntegrationTest, CheckConsistencyIntegrationTest) {
+  using GC = bigtable::GcRule;
+  using namespace google::cloud::testing_util::chrono_literals;
+
+  std::string id =
+      "it-" + google::cloud::internal::Sample(
+                  generator_, 8, "abcdefghijklmnopqrstuvwxyz0123456789");
+  auto project_id = bigtable::testing::TableTestEnvironment::project_id();
+
+  auto instance_admin_client = bigtable::CreateDefaultInstanceAdminClient(
+      project_id, bigtable::ClientOptions());
+  bigtable::InstanceAdmin instance_admin(instance_admin_client);
+
+  bigtable::InstanceId instance_id(id);
+  bigtable::DisplayName display_name("Integration Tests " + id);
+
+  // Replication needs at least two clusters
+  auto cluster_config_1 =
```

Comment:
If you are going to create instances dynamically we should not hardcode the zones, is there a way we can get them from the command-line or something?

Reply:
we may create a common helper class and define possible zones there, test case will request this helper class and get back zones in `std::vector`. This way hard coding is limited to one helper class and test cases are self sufficient.

Reply:
I would really prefer if we got these from the command-line, or in the worst case scenario from environment variables.  The driver scripts are much easier places to change these things.

Reply:
Fixed the code by passing zones as arguments.